src/main/scala/li/cil/oc/server/component/DebugCard.scala

```diff
@@ -388,6 +388,16 @@ object DebugCard {
       null
     }
 
+    @Callback(doc = """function(x:number, y:number, z:number, sound:string, range:number) -- Play a sound at the specified coordinates.""")
+    def playSoundAt(context: Context, args: Arguments): Array[AnyRef] = {
+      checkEnabled()
+      val (x, y, z) = (args.checkInteger(0), args.checkInteger(1), args.checkInteger(2))
+      val sound = args.checkString(3)
+      val range = args.checkInteger(4)
+      world.playSoundEffect(x, y, z, sound, range / 15 + 0.5F, 1.0F)
```

Comment:
It's "Range" Because if you give it a large number it plays for a large range.  I've used numbers as high as 400 to reach very far away from the source block.  This is handy for playing a sound across an entire world, or atleast a vary large chunk of it.